Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:14.04
+MAINTAINER Shane Canon <scanon@lbl.gov>
+
+#
+# Install build dependencies and openjdk-8
+#
+RUN apt-get update && apt-get -y install git ant make gcc wget software-properties-common unzip && \
+   add-apt-repository -y ppa:openjdk-r/ppa && \
```

Comment:
why gcc, make, unzip? They should be installed automatically, right?
Actually, why gcc at all?